deployConfig/poolConfig.ts

```diff
@@ -53,44 +53,12 @@ export interface poolConfig {
 }
 
 export const poolConfigs: { [name: string]: poolConfig } = {
-  borrow_aETH_v1: {
```

Comment:
This is still active right?

Reply:
Still active to end of year, but not the latest config. Do you think we should be keeping the config for still-active-but-not-the-most-up-to-date IRS instances? Happy to keep if we think we will use it but I wasn't sure what we would use it for.

Reply:
yes, I'd keep this until the end of year and we can delete then. Just for consistency.